Web/Edubase.Web.UI/Assets/Scripts/Entry/merge-amalgamate.js

```diff
@@ -425,7 +425,7 @@ const mergersApp = new Vue({
 
       if (this.mergeDateError) {
         this.errors.push({
-          href: '#amalgamationdate-day',
+          href: '#amalgamation-date-day',
```

Comment:
@JaswiniP should the value be changed here as we are only changing ids? FYI @RogerHowellDfE

Reply:
An `href` with an `#` in it is (usually) a link to the HTML element with that `id` on the page (similar to CSS selectors). 

If updating the ID, it makes sense to also update links which point to it so looks good here :)